src/com/raghav/javabasics/day4/RemoveWhiteSpacesInString.java

```diff
@@ -0,0 +1,29 @@
+package com.raghav.javabasics.day4;
+//Write a java program to remove all the white-spaces in the String.
+
+public class RemoveWhiteSpacesInString {
+    public static void main(String args[]){
+        RemoveWhiteSpacesInString result = new RemoveWhiteSpacesInString();
+        result.removeSpaces("Write a java program, to remove all the white-spaces in the String");
```

Comment:
Output:
 W r i t e a j a v a p r o g r a m , t o r e m o v e a l l t h e w h i t e - s p a c e s i n t h e S t r i n g
Array is Empty/Null
Array is Empty/Null